scripts/generic-ci-tests.sh

```diff
@@ -108,7 +108,10 @@ case "$TEST_SUITE" in
                 paver test_system -s lms --extra_args="--attr='shard_2' --with-flaky" --cov_args="-p"
                 ;;
             "3")
-                paver test_system -s lms --extra_args="--attr='shard_1=False,shard_2=False' --with-flaky" --cov_args="-p"
+                paver test_system -s lms --extra_args="--attr='shard_3' --with-flaky" --cov_args="-p"
```

Comment:
Nit: This could be easier to update if we used something like:

  [1-3])
    paver test_system -s lms --extra_args="--attr='shard_$SHARD' --with-flaky" --cov_args="-p"
  4)
    paver test_system -s lms --extra_args="--attr='shard_1=False,shard_2=False,shard_3=False' --with-flaky" --cov_args="-p"

It would get even easier to update if we used the attr=value form for our attributes: `@attr(shard=1)`, because then we could have most of the shards use `--attr shard=N` and have the last shard use `--attr '!shard'`.